packages/test/test-end-to-end-tests/src/test/orderSequentially.spec.ts

```diff
@@ -185,12 +185,8 @@ describeCompat("Multiple DDS orderSequentially", "NoCompat", (getTestObjectProvi
 						type: "rejoin",
 					});
 				});
-			} catch (err) {
-				error = err as Error;
-			}
+			}, validateAssertionError("Unexpected message type submitted in Staging Mode"));
 
-			assert(error !== undefined, "No error");
-			validateAssertionError(error, "Unexpected message type submitted in Staging Mode");
```

Comment:
this use was a little different, but for no reason I could see, so I made it use the conventional style. If the callback was async it might make sense this way, but its not.